src/applications/disability-benefits/all-claims/pages/privateMedicalRecords.js

```diff
@@ -97,7 +97,7 @@ export const schema = {
         },
       },
     },
-    privateMedicalRecords: privateMedicalRecordAttachments,
+    privateMedicalRecordAttachments,
```

Comment:
I think we'll want to remove this property from the schema (in `vets-json-schema`) completely, so that there's only one `attachments` schema property in there. Can do that in a separate PR. Then, this page property can import the `attachments` property and modify the `enum` and `enumNames` for it to suit each upload page (assuming each upload page uses some subset of the enums in the `attachments` list.